app/models/season/phase_switcher.rb

```diff
@@ -1,44 +1,61 @@
+# frozen_string_literal: true
+
 class Season::PhaseSwitcher
 
+  WHITELISTED_PHASES = %i[
```

Comment:
👍  but please switch to 2-spaces wide soft tabs - the 4 spaces thing is Python only 😉 
It's just a Ruby Community best practise to go with 2 spaces.

Plus: I prefer the `i%(` over the `i[[` syntax, since it makes it more obvious that what's being invoked here is actually a function... Aside from that, it's also handy to freeze the constant afterwards:
```ruby
PHASES = %i(
    fake_proposals_phase
    fake_application_phase
    fake_coding_phase
    back_to_reality
).freeze
```

And on naming: I think it's perfectly fine to reduce this to `PHASES`, no need to prepend "whitelist" if there no such thing as general `PHASES` or "blacklisted" phases there.